narwhals/utils.py

```diff
@@ -2000,3 +2003,35 @@ def to_tuple(self) -> tuple[_T, ...]:
         # Collect and return as a `tuple`.
         it = self._into_iter()
         return it if isinstance(it, tuple) else tuple(it)
+
+
+def parse_interval_string(every: str) -> tuple[str, str]:
```

Comment:
could we return the number as an int here, so the type is `tuple[int, str]`, and then we don't need to call `int(` on the result each time?